### DISCUSSION TITLE
want to make a PR with autcommands where should I look?

### DISCUSSION THREAD

**Discussion by txtyash** [LunarVim/LunarVim#2883] [General]
I want to make a PR to add some autocommands to lunarvim. Here's what I did:
1. Cloned the repo(rolling)
2. edited LunarVim/lua/lvim/core/autocmds.lua
3. added the autocommands
4. ran the installer
5. opened lunarvim
6. tested the autocommands: didn't work
7. navigated to .local/share/lunarvim/lvim/lua/lvim/core/autocmds.lua
8. That file didn't have the changes i made in the cloned repo i ran the installer from
9. Making changes to this file `~/.local/share/lunarvim/lvim/lua/lvim/core/autocmds.lua` made the autocommands work in lunarvim as desired.
10. Now I need someone to point me to the file that I need to edit in the git repo such that after installation the autocommands work without tampering the file under .local/share.

**Comment by kylo252** [comment]
you can make changes directly in your current lunarvim installation, if you haven't modified anything in the installer, then it should be under `~/.local/share/lunarvim/lvim`

I recommend using [`gh`](https://cli.github.com)
```sh
cd ~/.local/share/lunarvim/lvim
gh repo fork
# make your changes
git add .
git commit -m "feat: more autocmds"
gh pr create
```
